docs/cudf/source/cudf_pandas/usage.md

```diff
@@ -56,7 +73,7 @@ with Pool(4) as pool:
     ...
 ```
 
-## Understanding performance - the `cudf.pandas` profiler
+## Profiling `cudf.pandas` Performance
```

Comment:
It's not really performance per se; we're profiling whether code runs on the CPU or GPU, with the implication that GPU usage is faster but really we don't provide that info.

If you feel strongly no need to change, just my 2c.
```suggestion
## Profiling `cudf.pandas`
```